### DISCUSSION TITLE
How to create job from database using quartz.net and c#

### DISCUSSION THREAD

**Discussion by karlosRivera** [quartznet/quartznet#453] [General] [answered]
suppose we have a cart when people purchase product from our web site then we push cart data in a sql server table with a column name isprocessded which has value 0 by default.

now i want to use quartz.net and c# to create job from database that when a new data will be added to cart with isprocessded =0 then a job will be created and job firing time will be set from database table.

i search google for this scenario but still found no good write up. so some one guide me with details instruction to handle this with quartz.net and c#. thanks

**Comment by lahma** [comment, marked as answer]
This would require quite an amount of coding. Probably would be easier to have polling jobs that monitor tables and rows, process them and update final status to the row.
